streamlit_folium/frontend/package.json

```diff
@@ -1,24 +1,24 @@
 {
-  "name": "streamlit_component_template",
-  "version": "0.1.0",
+  "name": "streamlit_folium",
+  "version": "0.2.0",
```

Comment:
I think this will be version 0.6 when it comes out, unless I'm misunderstanding what package.json refers to...does it refer to the Python version of this package or the version of the template?

Reply:
It refers to the version of the npm package that we're theoretically creating. Probably does make sense to keep the version the same as the python package, but I don't think it really matters.